components/button/button-move.js

```diff
@@ -113,7 +113,7 @@ class ButtonMove extends ThemeMixin(FocusMixin(RtlMixin(LitElement))) {
 					flex-direction: column;
 					gap: 2px;
 					margin: 0;
-					min-height: 1.8rem;
+					min-height: auto;
```

Comment:
So this was overriding the styles inherited from `buttonStyles` that gave it a `min-height` of `calc(2rem + 2px)`. Setting it to `1.8rem` _works_, since the two `d2l-icon`s (`0.85rem` each) plus the `2px` gap adds to `36px` which is the same as `1.8rem`. But using `auto` also works and seemed more future-proof to changes in the icon heights.